src/main/java/net/glease/ggfab/mte/MTE_AdvAssLine.java

```diff
@@ -911,7 +942,8 @@ public boolean canStart() {
         }
 
         public boolean hasInput() {
-            ItemStack stack = mInputBusses.get(id).getStackInSlot(0);
+            ItemStack stack = getInputBusContent(id);
+            if (stack == null) return false;
```

Comment:
you should also check if it equals NULL.

Reply:
getInputBusContent will never return NULL.